server/internal/adapter/integration/convert.go

```diff
@@ -253,3 +255,40 @@ func toModelSort(sort *integrationapi.SortParam, dir *integrationapi.SortDirPara
 func fromCondition(_ schema.Package, condition integrationapi.Condition) *view.Condition {
 	return condition.Into()
 }
+
+func fromRequestRoles(roles []integrationapi.ProjectRequestRole) []workspace.Role {
+	return lo.FilterMap(roles, func(r integrationapi.ProjectRequestRole, _ int) (workspace.Role, bool) {
+		if role := fromRequestRole(r); role != nil {
+			return *role, true
+		}
+		return workspace.Role(""), false
+	})
+}
```

Comment:
this implementation will ignore the wrong values! its better to return error in case user sent a not valid value